pal/minigame/taobao.ts

```diff
@@ -42,6 +51,18 @@ Object.defineProperty(minigame, 'orientation', {
 });
 // #endregion SystemInfo
 
+// @ts-expect-error TODO: move into minigame.d.ts
+minigame.testSupportLandscape = function () {
```

Comment:
- What does the todo mean?
- testSupportLandscape -> isSupportLandscape ?

Reply:
* Refer to the comments in the WeChat adapter, which means that this method should be encapsulated in minigame in the future.
* OK. `isSupportLandscape` maybe better.